rs/tests/consensus/backup/common.rs

```diff
@@ -64,48 +63,91 @@ use std::{
     path::{Path, PathBuf},
     process::{Command, Stdio},
 };
-use std::{fs::File, time::Duration};
 
 const DKG_INTERVAL: u64 = 9;
 const SUBNET_SIZE: usize = 4;
 const DIVERGENCE_LOG_STR: &str = "The state hash of the CUP at height ";
 
-fn main() -> Result<()> {
-    SystemTestGroup::new()
-        .with_setup(config)
-        .with_timeout_per_test(Duration::from_secs(15 * 60))
-        .add_test(systest!(test))
-        .execute_from_args()?;
+fn config_common() -> InternetComputer {
```

Comment:
nit: I think `setup_common` would more idiomatic name fo this function

Reply:
Good point 👍